features/step_definitions/kata.rb

```diff
@@ -166,7 +166,12 @@
     logger.info("There's already an existing 'kataconfig' resuing it...")
     project(kata_ns)
     step %Q/I switch to cluster admin pseudo user/
+    # make sure kata-operator is running first before installing the kataconfig
+    step %Q/a pod is present with labels:/, table(%{
```

Comment:
What if the pod is present but in a bad state?

Reply:
@jensfr then the step will fail and test will mark as such.

Reply:
@jensfr I changed it to use a step to wait for the pod to become `ready` instead...